File: 프로그래머스/0/120844. 배열 회전시키기/배열 회전시키기.js
function solution(numbers, direction) {
    var answer = [];
    
    if(direction === "right"){
        const e = numbers.pop();
        numbers.unshift(e);
    }
    else{
        const e = numbers.shift();
        numbers.push(e);
    }
    
    return numbers;
}

File: 프로그래머스/0/120844. 배열 회전시키기/배열 회전시키기.java
import java.util.ArrayList;
import java.util.List;

class Solution {
    public int[] solution(int[] numbers, String direction) {
        
        List<Integer> list = new ArrayList<>();
        for(var i : numbers)
            list.add(Integer.valueOf(i));
        
        if(direction.equals("right")){
            list.add(0, list.remove(list.size() - 1));
        }
        else{
            list.add(list.remove(0));
        }
        
        
        return list.stream()
            .mapToInt(Integer::valueOf)
            .toArray();
        
    }
}